components/brave_webcompat_reporter_ui/brave_webcompat_reporter.tsx

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import { render } from 'react-dom'
+import { Provider } from 'react-redux'
+import { bindActionCreators } from 'redux'
+
+import defaultTheme from 'brave-ui/theme/brave-default'
+import { ThemeProvider } from 'brave-ui/theme'
+require('../fonts/muli.css')
+require('../fonts/poppins.css')
+
+// Components
+import App from './screens/app'
+
+// Utils
+import store from './store'
+import * as webcompatReporterActions from './actions/webcompatreporter_actions'
+
+let actions: any
+
+window.cr.define('brave_webcompat_reporter', function () {
+  'use strict'
+
+  function loadDialogArgs () {
+    const dialogArgsRaw = chrome.getVariableValue('dialogArguments')
+    let dialogArgs
+    try {
+      dialogArgs = JSON.parse(dialogArgsRaw)
+    } catch (e) {
+      console.error('Error parsing incoming dialog args: ', dialogArgsRaw, e)
+    }
+
+    getActions().setSiteUrl(dialogArgs.siteUrl)
+  }
+
+  function initialize () {
+    loadDialogArgs()
+
+    render(
+      <Provider store={store}>
+        <ThemeProvider theme={defaultTheme}>
```

Comment:
Since this is a new dialog, a relatively small UI, and everything else non-rewards is dark-mode aware, can we see if it already by-default supports dark mode in all its color choices, and if not look to add theme properties as needed?
NTP gives example for using the default dark and light themes https://github.com/brave/brave-core/blob/master/components/brave_new_tab_ui/brave_new_tab.tsx#L32

Reply:
Hmm, I remember why I didn't originally support theming - I'm getting this error when I try to call `chrome.braveTheme.getBraveThemeType`:
```
TypeError: Cannot read property 'getBraveThemeType' of undefined
```
Any ideas? Do I need to import something outside of the JS to get this to work?

Reply:
Yes you need to give your new page permission to the `chrome.braveTheme` API

Reply:
That works for me.

Rather than rebuilding another panel-specific theme layer, any chance I can get some basic colors loaded into `brave-dark`? The component types I'm using here are super basic.

Reply:
https://github.com/brave/brave-ui/pull/542 allows the panel to be themed in dark mode.